src/braket/ocean_plugin/braket_sampler.py

```diff
@@ -175,15 +175,29 @@ def sample_ising(
 
         Examples:
             This example submits a two-variable Ising problem mapped directly to qubits
-            0 and 1.
+            0 and 1 on a D-Wave 2000Q device.
 
             >>> from braket.ocean_plugin import BraketSampler
-            >>> sampler = BraketSampler(s3_destination_folder, "device_arn_1")
-            >>> sampleset = sampler.sample_ising({0: -1, 1: 1}, {}, resultFormat="HISTOGRAM")
+            >>> device_arn_1 = "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6"
+            >>> sampler = BraketSampler(s3_destination_folder, device_arn_1)
+            >>> Q = {0: -1, 1: 1}
```

Comment:
The ising examples should use `h` instead of `Q` to match the parameter names. Also the first parameter is the linear term, so `Q` could be misleading to someone expecting a quadratic term.